__init__.py

```diff
@@ -63,6 +65,7 @@
 gameEditorEnum = (
     ("SM64", "SM64", "Super Mario 64"),
     ("OOT", "OOT", "Ocarina Of Time"),
+    ("MK64", "MK64", "Mario Kart 64"),
     ("Homebrew", "Homebrew", "Homebrew"),
 )
```

Comment:
One issue here is that blender actually stores enum values as integers, so if someone had homebrew it would be changed, you might be able to get around this, but I'd need confirmation (@Dragorn421 do you know if this is correct?)
```suggestion
gameEditorEnum = (
    ("SM64", "SM64", "Super Mario 64", 0),
    ("OOT", "OOT", "Ocarina Of Time", 1),
    ("MK64", "MK64", "Mario Kart 64", 3),
    ("Homebrew", "Homebrew", "Homebrew", 2),
)
```

Reply:
or I put MK64 after homebrew ?

Reply:
That would probably be a bit weird I think, but I just tested this and it works